src/tron/tronEngine.ts

```diff
@@ -527,6 +530,76 @@ export class TronEngine extends CurrencyEngine<TronTools> {
         this.addTransaction(currencyCode, edgeTransaction)
         return out
       }
+
+      // Parse freeze transactions
+      const freezeTransaction = asMaybe(asFreezeBalanceContract)(contract)
+      if (freezeTransaction != null) {
+        const {
+          parameter: {
+            value: { owner_address: fromAddress, frozen_balance: frozenAmount }
+          }
+        } = freezeTransaction
+
+        if (
+          hexToBase58Address(fromAddress) !== this.walletLocalData.publicKey
+        ) {
+          break
+        }
+
+        const feeNativeAmount = retArray[0].fee.toString()
+        const nativeAmount = add(frozenAmount.toString(), feeNativeAmount)
+        const currencyCode = this.currencyInfo.currencyCode
```

Comment:
`const {currencyCode} = this.currencyInfo`

Reply:
I found a few more of these so they're all fixed in the fixup